### ISSUE TITLE
Difficulty with composing rules and accessibility requirements mapping

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act-rules#108]
The new requirement that implementations need to have a consistent accessibility requirements mapping with the rule. This creates some problems for composing rules (atomic rules used in composite rules). 

For example on [Document has heading for non-repeated content](https://www.w3.org/WAI/standards-guidelines/act/rules/047fe0/proposed/),  this rule does not map to any WCAG success criteria, because having headings on repeated blocks isn't required for conformance to 2.4.1. However, all the passed exampled are cases where 2.4.1 is satisfied, and all the failed examples are examples where 2.4.1 is failed.

An implementation reporting 2.4.1 failures on those failed examples is absolutely right to do so. It's kind of odd therefor that we wouldn't consider that consistent. I haven't fully grocked this problem yet, but there I think generally, I think it should be true that if an implementation is consistent with the composite rule, it shouldn't be inconsistent with its atomic rules.

**Comment by WilcoFiers** [comment]
Perhaps one thing we can do in the short term until we've properly figured this out is say that if a rule has no accessibility requirements at all, the "consistent with accessibility requirements" thing does not apply.
